.github/workflows/check-links.yml

```diff
@@ -3,7 +3,7 @@ name: Check for dead links
 on: 
   push:
     branches:
-      - main
+      - master
   workflow_dispatch:
```

Comment:
Good catch. We should migrate the use of 'master' to 'main' in all occurrences anyways